src/app/admin/components/courthouses/courthouses.component.html

```diff
@@ -7,5 +7,5 @@
 <app-courthouse-search-form (submitForm)="onSubmit($event)" (clear)="onClear()"></app-courthouse-search-form>
 
 @if (results$ | async; as results) {
-  <app-courthouse-search-results [results]="results" />
+  <app-courthouse-search-results [results]="results" [isFormEmpty]="isFormEmpty" />
```

Comment:
Rather than introducing an additional flag, you could use what's already there:

`<app-courthouse-search-results [results]="results" [show]="(isSubmitted$ | async) ?? false" />`